src/govuk/components/accordion/_index.scss

```diff
@@ -44,74 +37,212 @@
     .govuk-accordion {
       // Border at the bottom of the whole accordion
-      border-bottom: 1px solid $govuk-border-colour;
+      border-bottom: $govuk-accordion-bottom-border-width solid $govuk-border-colour;
     }
 
     // Borders between accordion sections
     .govuk-accordion__section {
-      padding-top: 0;
-    }
-
-    // Hide the body of collapsed sections
-    .govuk-accordion__section-content {
-      display: none;
-      @include govuk-responsive-padding(3, "top");
-      @include govuk-responsive-padding(3, "bottom");
-    }
-
-    // Show the body of expanded sections
-    .govuk-accordion__section--expanded .govuk-accordion__section-content {
-      display: block;
+      padding: 0;
     }
 
-    // This is styled to look like a link not a button
-    .govuk-accordion__open-all {
-      @include govuk-font($size: 16);
+    .govuk-accordion__show-all {
+      @include govuk-font($size: 19);
       position: relative;
       z-index: 1;
-      margin: 0;
-      padding: 0;
+
+      margin: 0 0 govuk-spacing(2) 0;
+      padding: 7px 2px 7px 0;
+
       border-width: 0;
+
       color: $govuk-link-colour;
       background: none;
+
       cursor: pointer;
       -webkit-appearance: none;
 
-      @include govuk-link-common;
-      @include govuk-link-style-default;
-
       // Remove default button focus outline in Firefox
       &::-moz-focus-inner {
         padding: 0;
         border: 0;
       }
+
+      &:hover {
+        color: $govuk-accordion-base-colour;
+        background: $govuk-accordion-hover-colour;
+
+        .govuk-accordion__section-toggle-text {
+          color: $govuk-accordion-base-colour;
+        }
+
+        .govuk-accordion-nav__chevron {
+          color: $govuk-accordion-base-colour;
+          background: $govuk-accordion-base-colour;
+        }
+
+        .govuk-accordion-nav__chevron:after {
+          color: $govuk-accordion-hover-colour;
+        }
+      }
+
+      &:focus {
+        @include govuk-focused-text;
+
+        .govuk-accordion-nav__chevron {
+          background: $govuk-accordion-base-colour;
+        }
+
+        .govuk-accordion-nav__chevron:after {
+          color: $govuk-accordion-icon-focus-colour;
+        }
+      }
+    }
+
+    .govuk-accordion__section-heading {
+      @include govuk-font($size: 24, $line-height: 1);
+      // Override browser defaults to ensure consistent element height
+      margin-top: 0; // Override browser default
+      margin-bottom: 0; // Override browser default
+    }
+
+    .govuk-accordion__section-heading-focus-wrapper {
+      margin-bottom: govuk-spacing(1);
+    }
+
+    // Create Chervon icon aligned with text
+    .govuk-accordion-nav__chevron {
+      box-sizing: border-box;
+      display: inline-block;
+
+      position: relative;
+
+      width: 20px;
+      height: 20px;
+
+      border: 1px solid;
+      border-radius: 50%;
+
+      vertical-align: text-top;
+
+      // IE8 fallback of icon
+      @include govuk-if-ie8 {
```

Comment:
The arrow lines for IE8 are very narrow (might not be visible to all users) and also slightly overlaps the borders of the box. We don't need to do anything too fancy here for IE8 but there might some slightly improvements we could make, I think @chris-gds is going to experiment with bolding the icon.

Reply:
Update to alter icon to improve this (Before / After) 

![IE8-update](https://user-images.githubusercontent.com/71266765/128318304-6df191b3-699f-4c53-8213-5983b76f437b.jpg)

(Possible action point: Design icon AC: include IE8)